lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
We'll need to run `apm publish --rename new-package-name`, as documented [here](https://atom.io/docs/v1.5.3/behind-atom-maintaining-your-packages#rename-your-package).